go/ct/driver/run.go

```diff
@@ -97,6 +97,14 @@ func doRun(context *cli.Context) error {
 	}
 
 	opRun := func(state *st.State) rlz.ConsumerResult {
+		result := rlz.ConsumeContinue
+		defer func() {
+			if r := recover(); r != nil {
+				result = rlz.ConsumeAbort
```

Comment:
I think this result over-ride only works for named return parameters. Right?

So you would have to change the signature to
```
func(state *st.State) (result rlz.ConsumerResult) {
```